core/templates/pages/exploration-player-page/services/answer-classification.service.spec.ts

```diff
@@ -35,7 +35,7 @@ import { TextInputRulesService } from 'interactions/TextInput/directives/text-in
 import { AlertsService } from 'services/alerts.service';
 import { TextInputPredictionService } from 'interactions/TextInput/text-input-prediction.service';
 
-describe('Answer Classification Service', () => {
+fdescribe('Answer Classification Service', () => {
```

Comment:
Please don't use test blockers here and elsewhere.

Reply:
Fixed.